views/peeps/timeline.erb

```diff
@@ -0,0 +1,8 @@
+  <h1> Welcome to Chitter </h1>
+
+  <a href="http://localhost:9292/peeps/new"><input type=button value='Post a Peep'></a>
```

Comment:
possibly better to use a form action for this like you did on peeps/new.erb. As if someone else runs rackup they might have a different port and it won't work